app/src/app/components/delete-dialog/delete-dialog.component.html

```diff
@@ -1,6 +1,7 @@
 <div class="delete-dialog">
     <h2>Are you sure you want to delete this object?</h2>
     <h4 *ngIf="hardDelete">WARNING: This object will be permanently deleted.</h4>
+    <h4 *ngIf="collectionDelete">WARNING: This will delete ALL objects associated with the collection.</h4>
```

Comment:
This warning should be updated to reflect the back-end change that's being made, where objects that exist in other collections will not be deleted:
```
WARNING: This action will delete all objects associated with this collection. If an object is referenced in another collection, it will not be deleted.
```